src/algebra/ordered_field.lean

```diff
@@ -655,4 +657,16 @@ by rw [abs_div, abs_one]
 lemma abs_inv (a : α) : abs a⁻¹ = (abs a)⁻¹ :=
 (abs_hom : monoid_with_zero_hom α α).map_inv' a
 
+-- TODO: add lemmas with `a⁻¹`.
+lemma one_div_strict_mono_decr_on : strict_mono_decr_on (λ x : α, 1 / x) (set.Ioi 0) :=
+λ x x1 y y1 xy, (one_div_lt_one_div (set.mem_Ioi.mp y1) (set.mem_Ioi.mp x1)).mpr xy
+
+lemma one_div_pow_mono (a1 : 1 ≤ a) : monotone (λ n : ℕ, order_dual.to_dual 1 / a ^ n) :=
```

Comment:
I think it's probably worth having the version without order_dual either instead or as well

Reply:
Eric, I just pushed a version with both, the `monotone` and the simple inequalities version: does this look good?